source/includes/steps-source-command-palette.yaml

```diff
@@ -0,0 +1,40 @@
+---
+title: "In |vscode-short|, open the Command Palette."
+level: 4
+ref: vsce-command-palette
+content: |
+
+  .. include:: /includes/list-tables/command-palette-options.rst
+---
+title: "From the Command Palette, choose where you want to save the 
```

Comment:
[nit] You can only do this once you actually click `Connect` to submit. Maybe include that as a step before this one